src/sage/rings/number_field/order.py

```diff
@@ -1179,6 +1179,39 @@ def __hash__(self):
         """
         return hash((self._K, self._module_rep))
 
+    def conductor(self):
+        r"""
+        For orders in *quadratic* number fields, return the conductor
+        of this order.
+
+        The conductor is the unique positive integer `f` such that
+        the discriminant of this order is `f^2` times the discriminant
+        of the containing quadratic field.
+
+        Not implemented for orders in number fields of degree `\neq2`.
+
+        .. SEEALSO ::
+
+            :meth:`sage.rings.number_field.number_field.NumberField_quadratic.order_of_conductor`
+
+        EXAMPLES::
+
+            sage: K.<t> = QuadraticField(-101)
+            sage: K.maximal_order().conductor()
+            1
+            sage: K.order(5*t).conductor()
+            5
+            sage: K.discriminant().factor()
+            -1 * 2^2 * 101
+            sage: K.order(5*t).discriminant().factor()
+            -1 * 2^2 * 5^2 * 101
+        """
+        if not isinstance(self._K, sage.rings.abc.NumberField_quadratic):
+            raise NotImplementedError('not implemented for number fields of degree != 2')
+        D = self.discriminant()
+        D0 = self._K.discriminant()
+        return ZZ(D / D0).sqrt()
```

Comment:
```suggestion
        return ZZ(D // D0).sqrt()
```
Perhaps even the `ZZ` call here becomes unnecessary?